Formula/opencsg.rb

```diff
@@ -0,0 +1,38 @@
+class Opencsg < Formula
+  desc "The CSG rendering library"
+  homepage "http://www.opencsg.org"
+  url "http://www.opencsg.org/OpenCSG-1.4.0.tar.gz"
+  sha256 "ecb46be54cfb8a338d2a9b62dec90ec8da6c769078c076f58147d4a6ba1c878d"
+
+  depends_on "qt5" => :build
+  depends_on "glew"
+
+  patch do
```

Comment:
Can you please add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!